<?xml version="1.0" ?>

<launch>

    <arg name="robot_ns" default="robot1" />

    <include file="$(find multi_sim)/launch/arlaunch.launch">
        <arg name="robot_ns" value="$(arg robot_ns)" />
    </include>

    <node name="ud_params" pkg="multi_sim" type="updateParams.py" output="screen"/>
   
</launch>